contracts/src/periphery/cert/router/EigenDACertVerifierRouter.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifier} from "src/periphery/cert/interfaces/IEigenDACertVerifier.sol";
+import {IEigenDACertVerifierRouter} from "src/periphery/cert/interfaces/IEigenDACertVerifierRouter.sol";
+import {OwnableUpgradeable} from "lib/openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+
+contract EigenDACertVerifierRouter is IEigenDACertVerifierRouter, OwnableUpgradeable {
+    mapping(uint32 => address) public certVerifiers;
+    uint32[] public certVerifierABNs;
+
+    event CertVerifierAdded(uint32 indexed abn, address indexed certVerifier);
+
+    error ABNNotInFuture(uint32 abn);
+    error ABNNotGreaterThanLast(uint32 abn);
+    error InvalidCertLength();
+    error NoCertVerifierAvailable();
+    error NoCertVerifierFound(uint32 rbn);
+
+    /// IEigenDACertVerifierRouter ///
+
+    function checkDACert(bytes calldata certBytes) external view returns (uint8) {
+        return IEigenDACertVerifier(getCertVerifierAt(_getRBN(certBytes))).checkDACert(certBytes);
+    }
+
+    function getCertVerifierAt(uint32 rbn) public view returns (address) {
+        return certVerifiers[_findClosestRegisteredABN(rbn)];
+    }
+
+    /// ADMIN ///
+
+    function initialize(address _initialOwner) external initializer {
+        _transferOwnership(_initialOwner);
+    }
+
+    function addCertVerifier(uint32 abn, address certVerifier) external onlyOwner {
+        if (abn <= block.number) {
```

Comment:
I don't see any consideration for the bootstrapping of the router.

When a new router is created, the user needs to be able to define a cert verifier address that will be used for all historical `rbn`s. Conceptually, that means adding an entry with an `abn` key of `0`, which isn't supported by this `addCertVerifier`.

There are two ways I can think of handling this bootstrapping:
1. If no cert verifiers are defined, permit a single address to be added with `abn=0`
2. Require / permit an address parameter during initialization, which is automatically added to cert verifier mapping, with key `0`. I think I personally prefer this strategy

Reply:
Done